raiden/network/transport/matrix/utils.py

```diff
@@ -328,70 +333,19 @@ def log(self) -> BoundLoggerLazyProxy:
         return self._log
 
 
-def join_broadcast_room(client: GMatrixClient, name: str, servers: Sequence[str] = ()) -> Room:
-    """Join or create a broadcast public room with given name
-
-    First, try to join room on own server (client-configured one)
-    If can't, try to join on each one of servers, and if able, alias it in our server
-    If still can't, create a public room with name in our server
+def join_broadcast_room(client: GMatrixClient, broadcast_room_alias: str) -> Room:
+    """ Join the public broadcast through the alias `broadcast_room_alias`.
 
-    Params:
-        client: matrix-python-sdk client instance
-        name: name or alias of the room (without #-prefix or server name suffix)
-        servers: optional: sequence of known/available servers to try to find the room in
-    Returns:
-        matrix's Room instance linked to client
+    When a new Matrix instance is deployed the broadcast room _must_ be created
+    and aliased, Raiden will not use a server that does not have the discovery
+    room properly set. Requiring the setup of the broadcast alias as part of
+    the server setup fixes a serious race condition where multiple discovery
+    rooms are created, which would break the presence checking.
```

Comment:
@ulope could you please add here a link to the issue for the presence problem? This context is important

Reply:
done